<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 235, 239);</string>
  </property>
  <widget class="QLabel" name="InformazioniProfilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Visita paziente</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDimetti">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
}
</string>
   </property>
   <property name="text">
    <string>Dimetti</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRicovera">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(0, 57, 97,0);
	color: rgb(230, 235, 239);
color: rgb(230, 235, 239);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
    }</string>
   </property>
   <property name="text">
    <string>Ricovera</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>290</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Codice di emergenza:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../MATERIALE/Back.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>210</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid rgba(230, 235, 239,0.6);
border-radius: 6px;
padding: 5px;
}
                            
QTextEdit:focus {
background-color: rgba(40, 40, 40,0.7);
}





QScrollBar:vertical {
background:rgba(0,0,0,0);
width: 5px;
margin: 0px 0 0px 0;


}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background-color: rgb(40, 40, 40);
border-radius: 2.2px;
}

QScrollBar::handle:vertical {
background-color: rgba(230, 235, 239,0.5);
min-height: 20px;
border-radius: 2px;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Prognosi</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoVisita">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>135</y>
     <width>220</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../MATERIALE/IconaVisitaomb.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Informazioni del paziente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>265</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Diagnosi</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>290</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 6px;
padding: 5px;
}
                            
QTextEdit:focus {
background-color: rgba(40, 40, 40,0.7);
}





QScrollBar:vertical {
background:rgba(0,0,0,0);
width: 5px;
margin: 0px 0 0px 0;


}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background-color: rgb(40, 40, 40);
border-radius: 2.2px;
}

QScrollBar::handle:vertical {
background-color: rgba(230, 235, 239,0.5);
min-height: 20px;
border-radius: 2px;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Inserisci Condizione del paziente</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cognome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CodiceDiEmergenza">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Back</zorder>
  <zorder>Sheda</zorder>
  <zorder>InformazioniProfilo</zorder>
  <zorder>ButtonDimetti</zorder>
  <zorder>ButtonRicovera</zorder>
  <zorder>label_3</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>LogoVisita</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>Cognome</zorder>
  <zorder>Nome</zorder>
  <zorder>CodiceDiEmergenza</zorder>
 </widget>
 <resources>
  <include location="../../../../../../asa.qrc"/>
 </resources>
 <connections/>
</ui>
